app/build.gradle

```diff
@@ -23,7 +23,7 @@ dependencies {
     implementation fileTree(dir: 'libs', include: ['*.jar'])
     implementation 'androidx.appcompat:appcompat:1.1.0'
     implementation 'androidx.constraintlayout:constraintlayout:1.1.3'
-    implementation project(path: ':squaremenu')
```

Comment:
@adityapp is it better to use `gridmenu` for the library name and change the sample app name to something else?